recipe/meta.yaml

```diff
@@ -24,22 +25,23 @@ requirements:
     - setuptools_scm >=3.4.3
```

Comment:
Thanks!

Reply:
The correct notation is:
```
 - setuptools_scm >=3.4.3
 - toml
```
I would _translate_ `setuptools_scm[toml]` as `setuptools_scm` including `toml`

Reply:
Interestingly enough, I think setuptools_scm means this:
https://github.com/pypa/setuptools_scm/blob/a5acb6845bf25ec9163020ea02d95e473051569b/setup.py#L53

This could of course be an oversight as well.